Libraries/LibC/time.h

```diff
@@ -43,7 +43,7 @@ struct tm {
     int tm_isdst; /* Daylight saving time */
 };
 
-extern long timezone;
+extern long timezone; // The difference in seconds between UTC and local time
```

Comment:
Let's not use C++ style comments ( `//` ) in C library headers.

Reply:
Fixed.